plaidml/op/op.h

```diff
@@ -320,6 +328,16 @@ inline edsl::Tensor elu(const edsl::Tensor& I, double alpha) {
   return details::op("elu", args).as_tensor();
 }
 
+inline edsl::Tensor explicit_padding(  //
+    const edsl::Tensor& x,             //
+    const std::vector<int>& lo_pads,   //
+    const std::vector<int>& hi_pads,   //
+    PadMode mode,                      //
+    const float& padval) {
```

Comment:
This should be `double` instead of `float`. (In the long run, we'll need to think about how to deal with `padval` since it's supposed to match the tensor's type, but `double` should be fine for now.)

Reply:
Also, C++ wise, there's really no reason to pass a POD (Plain-Old-Datatype) by-const-ref (like `const float&`). Passing by value is usually easier to read and doesn't cause any performance hit.